OutOfSchool/OutOfSchool.DataAccess/Extensions/EntityEntryExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace OutOfSchool.Services.Extensions
+{
+    public static class EntityEntryExtensions
+    {
+        public static string GetTableName(this EntityEntry entry) =>
+            entry.Metadata.GetAnnotation("Relational:TableName").Value.ToString();
```

Comment:
Can you use `entry.Metadata.GetTableName()` here? Or at least you're able to use RelationalAnnotationNames.TableName instead of string

Reply:
Good point. But this method is no longer used, so it will be removed.